alignak/objects/service.py

```diff
@@ -301,6 +301,14 @@ def get_name(self):
             return self.name
         return 'SERVICE-DESCRIPTION-MISSING'
 
+    def get_servicegroups(self):
```

Comment:
We can remove this function by replace _get_servicegroups()_ calls by self.servicegroups